Content.Shared/Humanoid/Markings/MarkingPrototype.cs

```diff
@@ -53,9 +58,12 @@ public sealed partial class MarkingPrototype : IPrototype
         [DataField("sprites", required: true)]
         public List<SpriteSpecifier> Sprites { get; private set; } = default!;
 
+        /// Might as well add that it's from Imp even if it's ONE line I guess?
+		public string? Shader { get; private set; } = null;
```

Comment:
Mb I meant to suggest this lmao
```suggestion
		[DataField]
		public string? Shader { get; private set; } = null;
```